scripts/img2imgalt.py

```diff
@@ -64,6 +78,13 @@ def find_noise_for_image(p, cond, uncond, cfg_scale, steps):
 
 # Based on changes suggested by briansemrau in https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/736
 def find_noise_for_image_sigma_adjustment(p, cond, uncond, cfg_scale, steps):
+    if shared.sd_model.is_sdxl:
+        cond_tensor = cond['crossattn']
+        uncond_tensor = uncond['crossattn']
+        cond_in = torch.cat([uncond_tensor, cond_tensor])
+    else:
+        cond_in = torch.cat([uncond, cond])
+

```

Comment:
also `cond_in` hear is unused
I think you might have put this block here by accident

Reply:
yeah, removed